src/sphinx_compas2_theme/__init__.py

```diff
@@ -57,49 +54,6 @@
 ]
 
 
-def get_linkcode_resolve(organization, repo):
```

Comment:
if this function doesn't exist anymore it will cause current builds to fail...

Reply:
perhaps readd a function with the same name that returns a partial

Reply:
fixing it...

Reply:
fixed